### ISSUE TITLE
[IOS-XE] show lisp instance-id {instance_id} ipv4 map-cache {prefix}

### ISSUE THREAD

**Issue by dmychepk** [CiscoTestAutomation/genieparser#884]
2024-08-29 18:08:54,159: %UNICON-INFO: +++ svs-fc-dis-r-1-1 with via 'ssh': executing command 'show lisp instance-id 1 ipv4 map-cache 0.0.0.0/0' +++
show lisp instance-id 1 ipv4 map-cache 0.0.0.0/0
LISP IPv4 Mapping Cache for LISP 0 EID-table default (IID 1), 1 entries

0.0.0.0/0, uptime: 00:58:48, expires: never, via static
  Sources: static
  State: forward-native, last modified: 00:58:48, map-source: local
  Exempt, Packets out: 503(201990 bytes), counters are not accurate (~ 00:00:12 ago)
  Configured as EID address space
  Encapsulating to proxy ETR 

SchemaMissingKeyError: Missing keys: [['lisp_id', 0, 'instance_id', 1, 'locators']]

**Comment by SohanTirpude** [comment]
Hello @dmychepk,

Thank you for letting us know about this issue. Kindly give me some time to debug this further.

Thank you.

**Comment by SohanTirpude** [comment]
Hello @dmychepk,

I found the solution for this issue and now after applying the solution the parsed output looks like this:
```
{
    "lisp_id": {
        0: {
            "instance_id": {
                1: {
                    "eid_table": "default",
                    "entries": 1,
                    "eid_prefix": "0.0.0.0/0",
                    "eid": "0.0.0.0",
                    "mask": 0,
                    "uptime": "00:58:48",
                    "expires": "never",
                    "via": "static",
                    "sources": "static",
                    "state": "forward-native",
                    "last_modified": "00:58:48",
                    "map_source": "local",
                    "activity": "Exempt",
                    "packets_out": 503,
                    "packets_out_bytes": 201990,
                    "counters_not_accurate": True
                }
            }
        }
    }
}
```
Can you please check and confirm if everything is correct or not?

Thank you.

**Comment by dmychepk** [comment]
looks good

**Comment by SohanTirpude** [comment]
Hello @dmychepk,

The PR got merged and the fix will be available in the upcoming pyats 24.9 version. Kindly try and let me know if you still see this issue with pyats 24.9 version.

Meanwhile, I am closing this ticket. If you are facing any other issues, kindly open a new ticket.

Thank you.
